AxonDeepSeg/params.py

```diff
@@ -0,0 +1,4 @@
+
+singe_mask_file_intensity = 255
+axon_intensity = 255
```

Comment:
instead of hard-coded 255, we can replace with `np.iinfo(np.uint8).max`
same for the 127 (make sure to round it properly)